pkgs/tools/misc/vmtouch/default.nix

```diff
@@ -2,14 +2,14 @@
 
 stdenv.mkDerivation rec {
   pname = "vmtouch";
-  version = "git-20150310";
+  version = "1.0.2";
```

Comment:
Unfortunately the `git` part of the name is the name. So now if you remove `git` anyone who had installed this tool by not attribute path will miss the update..